scripts/deps-csv.py

```diff
@@ -1,4 +1,8 @@
-#!/usr/bin/python3
```

Comment:
I took liberty to drop the shebang here cause we don't run python2 anywhere afaik